src/applications/static-pages/sass/modules/_m-facilities.scss

```diff
@@ -40,5 +40,140 @@ $duo-tone-lighten: #000000;
 }
 
 .facility-satisfaction-tile {
-  width: 147px;
+  width: 50%;
+
+  @include media($medium-large-screen) {
+    width: 147px;
+  }
+}
+
+.basic-facilities-list {
+  .facility {
+    &:last-child {
+      margin-bottom: 24px;
+    }
+  }
+}
+
+.region-img,
+.news-img {
+  width: 100%;
+  height: 165px;
+  object-fit: cover;
+
+  @include media($small-screen) {
+    height: 180px;
+    width: 342px;
+  }
+
+  @include media($medium-large-screen) {
+    height: initial;
+    width: initial;
+    object-fit: initial;
+  }
+}
+
+.facility-img {
+  width: 100%;
+  height: 164px;
+  object-fit: cover;
+
+  @include media($small-screen) {
+    height: 180px;
+    width: 342px;
+  }
+
+  @include media($medium-large-screen) {
+    height: initial;
+    width: initial;
+    object-fit: initial;
+  }
+}
+
+.news-img {
+  @include media($small-screen) {
+    height: 180px;
+    width: 100%;
+  }
+
+  @include media($medium-large-screen) {
+    height: initial;
+    width: initial;
+  }
+}
+
+.story-detail-img {
+  width: 100%;
+  height: 164px;
+  object-fit: cover;
+
+  @include media($medium-large-screen) {
+    width: 100%;
+    height: 295px;
+    object-fit: cover;
+  }
+}
+
+.event-detail-img {
+  width: 100%;
+  height: 159px;
+  object-fit: cover;
+
+  @include media($medium-large-screen) {
+    width: 100%;
+    height: 195px;
+    object-fit: cover;
+  }
+}
+
+.region-list {
+  .region-grid {
```

Comment:
Do you need this nesting? Do these styles work if they're not nested underneath `.region-list`?

Reply:
right! i can remove that :smile: